extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -316,7 +316,7 @@ class MembershipsButtonEdit extends Component {
 	render = () => {
 		const { className, notices } = this.props;
 		const { connected, connectURL, products } = this.state;
-
+		const align = this.props.attributes.align;
```

Comment:
Total nit but how about grouping these since we're already pulling from `this.props` in above line;
```js
const { attributes, className, notices } = this.props;
const { align } = attributes;
```

Reply:
Done!